camdecmpswks/functions/get-qatest-feedback.sql

```diff
@@ -1,6 +1,6 @@
-DROP FUNCTION IF EXISTS camdecmpswks.get_qatest_feedback(text) CASCADE;
+DROP FUNCTION IF EXISTS camdecmpswks.get_qatest_feedback(text[]) CASCADE;
 
-CREATE OR REPLACE FUNCTION camdecmpswks.get_qatest_feedback(testId text)
+CREATE OR REPLACE FUNCTION camdecmpswks.get_qatest_feedback(testId text[])
```

Comment:
For future reference, I would change the parameter name to indicate it has multiple values, not just a single value.  For instance, testIds or testIdList or testIdArray.